fapi-client/src/main/scala/com/gu/facia/api/utils/ExternalLinks.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.facia.api.utils
+
+import java.net.URI
+
+import scala.util.Try
+
+object ExternalLinks {
+  val corsOrigins = List(
+    "http://www.theguardian.com",
+    "https://www.theguardian.com",
+    "https://profile.theguardian.com",
+    "https://composer.gutools.co.uk",
+    "https://composer.release.dev-gutools.co.uk",
+    "https://composer.code.dev-gutools.co.uk",
+    "http://preview.gutools.co.uk",
+    "https://preview.gutools.co.uk")
+
+  val GuardianDomains = corsOrigins flatMap { uri =>
+    Try {
+      new URI(uri).getHost.stripPrefix("www.")
+    }.toOption
+  }
+
+  def external(url: String) = Try(Option(new URI(url).getHost).exists({ host => !GuardianDomains.exists({ domain =>
```

Comment:
This is a very long line!

Might also be clearer with a type annotation (or if it is renamed to `isExternal` it'll be clearer this is just a test returning a Bool? (with all the `Try` indirection it isn't at all clear from first glance what this does)